core/tests/ledger_cleanup.rs

```diff
@@ -502,14 +500,10 @@ mod tests {
                                 break;
                             }
                         } else {
-                            let mut slot_id = start_slot;
+                            let slot_id = start_slot;
                             if slot_id > 0 {
-                                for shred in shreds_with_parent.iter_mut() {
-                                    shred.set_slot(slot_id);
```

Comment:
The shred insertion benchmark code here is to reuse the shreds generated at line 482 by updating them in place to avoid re-generating shreds, which is shown to be slower than shred insertion.